cmd/logbridge/main.go

```diff
@@ -116,8 +116,23 @@ func startReader(ch chan<- []map[string]interface{}) error {
 	if err != nil {
 		return err
 	}
-
-	cursor, err := readCursor()
+	var cursor string
+	cursor, err = readCursor()
+	if err != nil {
+		return err
+	}
+	// check if cursor file is valid
+	// https://godoc.org/github.com/coreos/go-systemd/sdjournal#Journal.GetCursor
+	if len(cursor) == 0 {
+		_, err := j.Next()
```

Comment:
off-by-one would erroneously skip the first entry in the journal?

Reply:
what is the right behaviour if we don't have a cursor and we can't get with a new one?
I was refering to this from:

> GetCursor gets the cursor of the last journal entry reeferenced by the last completed Next/Previous function call. To call GetCursor, you must first have called one of the Next/Previous functions.

So if GetCursor fails, we call next and try again?